content/birds/victor.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Victor Allen
+sub: Product designer and developer
+bird: victor
+date: 2025-01-31
```

Comment:
I think this should be the date you joined the team, rather than the date of adding the page.